admin/workspace-management/process-logging.md

```diff
@@ -66,9 +66,24 @@ cloud provider's log viewer, or you can use `kubectl` to print the logs:
 
 ```bash
 kubectl logs \
-  -l "com.coder.username=zac" \        # Filter by the user "zac"
-  -l "com.coder.workspace.name=code" \ # Filter by the workspace "code"
-  -c exectrace                         # Only show logs from the sidecar
+  -l "com.coder.username=admin" \        # Filter by the user "admin"
```

Comment:
mainly a personal preference, for docs and scripts (as opposed to typing on the command-line), it seems clearer to use long names for options (e.g. `kubectl logs --selector="com.coder.username=admin"`). it's helpful for people less familiar with the command and its options